react/src/components/QrReaderOverlay/QrReaderOverlay.tsx

```diff
@@ -203,17 +224,24 @@ function QrReaderOverlay({
             </HStack>
             <HStack borderColor="blackAlpha.100" borderWidth={2} p={4} my={5}>
               <Text fontWeight="bold">By Label</Text>
-              <Input
-                type="string"
-                width={150}
-                onChange={(e) => setBoxLabelInputValue(e.currentTarget.value)}
-                value={boxLabelInputValue}
-              />
+              <FormControl isInvalid={!!boxLabelInputError}>
+                <Input
+                  type="string"
+                  width={150}
+                  onChange={(e) => onBoxLabelInputChange(e.currentTarget.value)}
+                  value={boxLabelInputValue}
+                />
+                <FormErrorMessage>{boxLabelInputError}</FormErrorMessage>
+              </FormControl>
               <Button
+                disabled={!!boxLabelInputError}
+                isLoading={isFindBoxByLabelForNonBulkModeLoading}
```

Comment:
I simulated a throttled network. When I click the button for the first time, a spinner appears and the input field turns empty. But it's still possible to quickly click the button a second time. Then I see the error message "Please enter a label id" (because of the now empty input) but I'm still being forwarded to the BoxView page. So it works overall but is slightly confusing to see the error message.

Reply:
@vahidbazzaz you said this behavior does not happen on your end, can you check again?

Reply:
@pylipp  Thanks, I'll look into it again

Reply:
@pylipp you are right! The behavior only occurs on slow networks.

Reply:
@HaGuesto @pylipp Just pushed the quick fix to disable the find button and input while it's processing

Reply:
confirming that this is fixed now. Input field and button are disabled while loading